source/aria.py

```diff
@@ -110,3 +110,10 @@ class AriaLivePoliteness(IntEnum):
 	OFF = 0
 	POLITE = 1
 	ASSERTIVE = 2
+
+	@classmethod
+	def parseFromString(cls, value: str):
+		try:
+			return next(v for v in cls if v._name_.lower() == value.lower())
+		except StopIteration:
+			raise ValueError(f"Unknown {cls.__qualname__}: {value}")

```

Comment:
You can just use a regular enum, with string values. Unless there is a reason to use an int that I'm missing. Note, I couldn't select the class line for this 'suggestion'

``` python
class AriaLivePoliteness(Enum)
	OFF = "off"
	POLITE = "polite"
	ASSERTIVE = "assertive"
```
Then in `nvdaControllerInternal_reportLiveRegion`

```python
politenessValue  = AriaLivePoliteness(value.lower())
```

EG in a repl: 
``` python
import enum
class AriaLivePoliteness(enum.Enum):
... 	OFF = "off"
... 	POLITE = "polite"
... 	ASSERTIVE = "assertive"
...

AriaLivePoliteness("off")
<AriaLivePoliteness.OFF: 'off'>
```

Reply:
See `def _get_liveRegionPoliteness(self):` in NVDAObjects/UIA/__init__.py:

`# Live setting enumeration values allign with aria.AriaLivePoliteness.`

SO, that's the reason why I initially chose for an int enum. If we change this, we'll need a mapping of values in UIAHandler. I'm ok with that, though. Just let me know whether you prefer one or the other :)